### PR TITLE
Feature/vue cli 3 transition

### PR DESCRIPTION
**Pull request by asvae**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,43 +5,44 @@
...
     "kewler": "^1.0.8",
     "leaflet-map": "^0.2.1",
     "medium-editor": "^5.23.3",
-    "normalize.css": "7.0.0",
+    "normalize.css": "8.0.0",
     "v-tooltip": "^2.0.0-rc.30",
-    "vee-validate": "2.0.9",
-    "vue": "2.5.2",
-    "vue-book": "0.0.3",
+    "vee-validate": "2.1.0-beta.7",
```
Comment (jean-moldovan) [inline package.json]: Why would want to use beta version?

File: src/sass/_utility-global.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Global utility classes.
+// NOTE Try you best not to use them. Seek a chance to destroy.
```
Comment (jean-moldovan) [inline src/sass/_utility-global.scss]: I don't think there is anything wrong with having a set of global css classes like:
`ellipsis`, `cursor-pointer` etc

File: src/vuestic-theme/vuestic-components/vuestic-chart/chart-types/DonutChart.js [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+      defaultOptions: {},
     }
   },
 
   mounted () {
     let options = Object.assign({}, DefaultOptions, this.defaultOptions, this.options)
     this.renderChart(this.chartData, options)
-  }
-})
+  },
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/vuestic-chart/chart-types/DonutChart.js]: Like to have comma?

**Comment by jean-moldovan** [comment]
Hello!

Are we going to be able to push this to staging first (when we have one)? We also need to make sure that this is in sync with other big pending PRs.

Thanks!

File: src/vuestic-theme/vuestic-components/vuestic-chart/chart-types/DonutChart.js (hunk 3 above)
Comment (asvae) [reply src/vuestic-theme/vuestic-components/vuestic-chart/chart-types/DonutChart.js]: Indeed. Commas are superb.

File: src/sass/_utility-global.scss (hunk 2 above)
Comment (asvae) [reply src/sass/_utility-global.scss]: @jean-moldovan 
We can provide a set of global classes to be used in app (for speed). Using them in components would be poor design choice though.

**Pull request merged**